crates/libs/result/src/error.rs

```diff
@@ -13,35 +14,54 @@ impl Error {
     pub const fn empty() -> Self {
         Self {
             code: HRESULT(0),
+            #[cfg(windows)]
             info: None,
         }
     }
 
     /// Creates a new error object, capturing the stack and other information about the
     /// point of failure.
     pub fn new<T: AsRef<str>>(code: HRESULT, message: T) -> Self {
```

Comment:
What about just having a different `new` method for non-Windows that lacks the `message` parameter?

Reply:
Probably similar to reasoning for `from_win32`